pom.xml

```diff
@@ -21,265 +27,56 @@
         <url>https://github.com/elucidation-project</url>
     </organization>
 
-    <developers>
```

Comment:
Why did you remove us?

Reply:
This will now get pulled from kiwi-parent.  Any contributors directly to this set of libraries can be added to the contributors section.